src/views/examples/Profile.js

```diff
@@ -257,7 +324,7 @@ const Profile = (props) => {
             <Button
               color="primary"
               href="#pablo"
-              onClick={printdata}
+              onClick={updateProfile}
```

Comment:
this is an action function, always name these with on as prefix like onUpdateProfile